src/pyfmi/fmi_util.pyx

```diff
@@ -371,16 +398,17 @@ cpdef convert_sorted_vars_name_desc(list sorted_vars):
         if desc_length_trial+1 > desc_length:
              desc_length = desc_length_trial + 1
 
-    name_output = <char*>FMIL.calloc((items+1)*name_length,sizeof(char))
+
+    name_output = <char*>FMIL.calloc((items+nof_diag_params+nof_diag_vars+1)*name_length,sizeof(char))
```

Comment:
Considering the number of times `items+nof_diag_params+nof_diag_vars+1` is repeated, perhaps it deserves to be in a local variable?

Reply:
Fixed.